packages/cells/package.json

```diff
@@ -54,10 +54,12 @@
         "@linaria/react": "^4.5.3",
         "@toast-ui/editor": "3.1.10",
         "@toast-ui/react-editor": "3.1.10",
+        "chroma-js": "^2.4.2",
```

Comment:
This dep is almost 1/3rd the total weight of GDG on its own. It looks like we want to use it for parsing, `brighten`, `darken`, `luminance`, and formatting.

`luminance` is an extremely simple calculation: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef

`brighten` is just darken with an inverted sign

`darken` is in fact the only complicated operator here because chroma-js will use the CIELAB colorspace to perform this computation. Conversion to/from lab is pretty expensive, but more importantly just takes a fair bit of code. Implementing an sRGB version of this would be the only tradeoff required and may produce results with sufficient fidelity to serve the purpose.

Converting to hex/css is a trivial operator.

Reply:
Sounds good. `darken` and `brighten` are not even really needed. I removed the usage of chroma-js and added the `getLuminance` and `toHex` from color2k to the `color-parser`